src/containers/toputilizers/components/TopUtilizersMultiHistogram.js

```diff
@@ -109,7 +108,7 @@ export default class TopUtilizersMultiHistogram extends React.Component {
       const utilizerNeighbors = neighbors.get(entityId) || [];
       const allEntityTypeIds = new Set(Object.keys(this.props.allEntityTypes));
       utilizerNeighbors.forEach((neighbor) => {
-        if (neighbor.has('neighborEntitySet') && neighbor.has('neighborDetails')) {
+        if (neighbor && neighbor.has('neighborEntitySet') && neighbor.has('neighborDetails')) {
```

Comment:
Weird... you shouldn't have to do this. You're doing `Immutable.fromJS()` in the reducer, so `neighbors` shouldn't have any null values (unless the backend is returning null).

What's the error that you're seeing?

An aside, on line 108, `utilizerNeighbors` has an inconsistent type. It'll either be `Immutable.Map`, or a regular JS array, which is not good. `immutable-js` allows you to pass a `notSetValue` for `.get()`, so you can do the following:

```
// utilizerNeighbors will be an empty Immutable.List if there's nothing found for .get(entityId)
const utilizerNeighbors = neighbors.get(entityId, Immutable.List);
```

https://facebook.github.io/immutable-js/docs/#/Map/get

Reply:
the backend is returning null. need to look into why that's happening too

Reply:
Ok, good to know. In the meantime, please make the change for `utilizerNeighbors`